src/lib/updating-element.ts

```diff
@@ -187,19 +190,29 @@ export abstract class UpdatingElement extends HTMLElement {
     // Allow user defined accessors by not replacing an existing own-property
     // accessor.
     if (this.prototype.hasOwnProperty(name)) {
-      return;
+      return null;
     }
+
     const key = typeof name === 'symbol' ? Symbol() : `__${name}`;
-    Object.defineProperty(this.prototype, name, {
-      get() { return this[key]; },
-      set(value) {
+
+    let defaultInitializer: null | (() => any) = descriptor ? descriptor!.initializer || null : null;
```

Comment:
This whole handling of the default value seems just a little messy but i can see why...

The default value **really** needs to be in the constructor but babel annoyingly doesn't put it there...